utils/FileHandler.cpp

```diff
@@ -0,0 +1,291 @@
+#include "FileHandler.hpp"
+
+namespace jbrush {
+
+Vector<Shape*> FileHandler::loadShapesFromDirectory(const QString& directory)
+{
+    // Get the file at the current directory
+    // and make a text stream to operate on it
+    QFile file(directory);
+    QTextStream qin(&file);
+
+    Vector<Shape*> shapesToReturn;  // Vector of shapes to be returned by the function
+    Shape* shapeExtracted;  // Shape extracted from the file
+    QString firstLine;  // Stores the first line of the file
```

Comment:
Use of this variable is far away from its declaration. They probably miss each other, and long distance relationships don't really work out :(